File: examples/read-inbox.mjs
import { ExpectboxAgent } from "expectbox-agents";
const agent = new ExpectboxAgent({
  apiKey: process.env.EXPECTBOX_AGENT_API_KEY,
});
const page = await agent.messages(process.env.EXPECTBOX_AGENT_INBOX_ID, {
  limit: 20,
});
console.log(`${page.items.length} messages; more: ${Boolean(page.next)}`);

File: examples/read_inbox.py
import os
from expectbox_agents import ExpectboxAgent

agent = ExpectboxAgent(os.environ['EXPECTBOX_AGENT_API_KEY'])
page = agent.messages(os.environ['EXPECTBOX_AGENT_INBOX_ID'], limit=20)
print(f"{len(page['items'])} messages; more: {bool(page['next'])}")
